imap/xapian_wrap.cpp

```diff
@@ -780,16 +780,14 @@ int xapian_dbw_open(const char **paths, xapian_dbw_t **dbwp, int mode, int nosyn
     }
 
     /* open the read-only databases */
-    if (mode == XAPIAN_DBW_XAPINDEXED) {
```

Comment:
Unfortunately this adds a significant IO hit :(  every time a message gets indexed you need to open all the databases, which is the reason why XapIndexed isn't on by default already despite being clearly the better option.

I think we're going to need to use conversations.db to see if there part is mentioned in an already indexed email :(

Reply:
Thanks for noticing this. OK, I'll rewrite it to use conversation.db.